packages/material-ui/src/internal/SelectableGroup/SelectableGroup.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as utils from './utils';
```

Comment:
What do you think of inlining the helper in this file? I'm not sure to see what we win at creating a single dependent module of 32 lines.